src/platform/imx8mp-verdin/desc.c

```diff
@@ -0,0 +1,38 @@
+/**
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) Bao Project and Contributors. All rights reserved.
+ */
+
+#include <platform.h>
+
+struct platform platform = {
+    .cpu_num = 4,
+    .region_num = 1,
+    .regions =  (struct mem_region[]) {
+        {
+            .base = 0x40000000,
+            .size = 0x80000000, // 2 GiB
+        },
+    },
+
+    .console = {
+        .base = 0x30880000,
+    },
+
+    .arch = {
+
+        .clusters =  {
+            .num = 1,
+            .core_num = (size_t[]) {4}
+        },
```

Comment:
```suggestion

```

No need for a single cluster if there is only one.

Unless mpidrs don't match the expected linearity (i.e., cpu 0 -> mpidr 0x0, cpu 1 -> mpidr 0x1, etc..)